windows/setup/workstation_setup.ps1

```diff
@@ -0,0 +1,54 @@
+$ErrorActionPreference = "Stop";
+$ProgressPreference="SilentlyContinue"
+
+trap { $host.SetShouldExit(1) }
+
+Invoke-WebRequest -UseBasicParsing "https://aka.ms/vs/15/release/vs_buildtools.exe" -OutFile "$env:TEMP\vs_buildtools.exe"
```

Comment:
FYI: For file downloads, `Start-BitsTransfer` is faster than `Invoke-WebRequest`. You may be able to use that.